stetho-sample/src/debug/java/com/facebook/stetho/sample/SampleDebugApplication.java

```diff
@@ -42,7 +49,56 @@ private void initializeStetho(final Context context) {
                 .finish();
           }
         })
-        .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(context))
+        .enableWebKitInspector(new ExtInspectorModulesProvider(context))
```

Comment:
Not sure why we need a separate class here.  It should be possible to write:

```java
.enableWebKitInspector(
  new DefaultInspectorModulesBuilder()
    .databaseDriver(createBlaBla())
    .finish());
```

This was the original intention of the new builder API at least.